tests/main.test.js

```diff
@@ -152,3 +152,23 @@ test("deliberately skipped .css shouldn't error", async () => {
   })
   expect(finalCss).toEqual('p{color:brown}')
 })
+
+test.skip('leaves used css variables', async () => {
+  const { finalCss } = await runMinimalcss('css-variables')
+  expect(finalCss).toMatch('--main-bg-color:')
+})
+
+test.skip('removes unused css variables', async () => {
+  const { finalCss } = await runMinimalcss('css-variables')
+  expect(finalCss).not.toMatch('--unused-color:')
```

Comment:
I think these two above tests test almost the same thing and thus they can be in the same test. 
E.g.
```javascript
test.skip('css variables', async () => {
  expect.assertions(2)
  const { finalCss } = await runMinimalcss('css-variables')
  expect(finalCss).toMatch('--main-bg-color:')
  expect(finalCss).not.toMatch('--unused-color:')
})